### PR TITLE
Added flow for borger.dk and added logic for multiple identities to h…

### PR DESCRIPTION
**Pull request by helmstedt**
…elpers

### CODE REVIEW

File: login_flows/e-boks.py [hunk 1]
```diff
@@ -51,34 +51,12 @@
...
-            selected_option = login_options[int(identity)-1].a["data-loginoptions"]
-            data["ChosenOptionJson"] = selected_option
-        else: 
-            raise Exception(f"Identity not in list of identities")
-    except:
-        raise Exception(f"Wrongly entered identity")
-    request = session.post(request.url, data=data)
-    soup = BeautifulSoup(request.text, features="html.parser")
+    request = choose_between_multiple_identitites(session, request, soup)
+    soup = BeautifulSoup(request.text, "xml")
```
Comment (Hundter) [inline login_flows/e-boks.py]: The parser should be lxml right?

File: login_flows/mitdk.py [hunk 2]
```diff
@@ -66,29 +66,8 @@
...
-            selected_option = login_options[int(identity)-1].a["data-loginoptions"]
-            data["ChosenOptionJson"] = selected_option
-        else: 
-            raise Exception(f"Identity not in list of identities")
-    except:
-        raise Exception(f"Wrongly entered identity")
-    request = session.post(request.url, data=data)
-    soup = BeautifulSoup(request.text, features="html.parser")
+    request = choose_between_multiple_identitites(session, request, soup)
+    soup = BeautifulSoup(request.text, "xml")
```
Comment (Hundter) [inline login_flows/mitdk.py]: The parser should be lxml right?

File: login_flows/borgerdk.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+args = argparser.parse_args()
+
+method, user_id, password, proxy = process_args(args)
+session = requests.Session()
+if proxy:
+    session.proxies.update({"http": f"socks5://{proxy}", "https": f"socks5://{proxy}" })
+
+# First part of borger.dk procedure
+request = session.get(f"https://www.borger.dk/mitoverblik")
+soup = BeautifulSoup(request.text, "xml")
```
Comment (Hundter) [inline login_flows/borgerdk.py:18]: The parser should be lxml right?

File: login_flows/borgerdk.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    "MitIDAuthCode": authorization_code
+}
+
+request = session.post("https://nemlog-in.mitid.dk/login/mitid", data=params)
+soup = BeautifulSoup(request.text, "lxml")
+
+# User has more than one login option
+if request.url == 'https://nemlog-in.mitid.dk/loginoption':
+    request = choose_between_multiple_identitites(session, request, soup)
+    soup = BeautifulSoup(request.text, "xml")
```
Comment (Hundter) [inline login_flows/borgerdk.py]: The parser should be lxml right?

**Review by Hundter — CHANGES_REQUESTED** [review]

File: login_flows/e-boks.py (hunk 1 above)
Comment (helmstedt) [reply login_flows/e-boks.py]: Yes, copy/paste error, fixed.

File: login_flows/mitdk.py (hunk 2 above)
Comment (helmstedt) [reply login_flows/mitdk.py]: Yes, copy/paste error, fixed.

File: login_flows/borgerdk.py (hunk 3 above)
Comment (helmstedt) [reply login_flows/borgerdk.py:18]: In this case, no! Cause of my copy/paste errors I think.

File: login_flows/borgerdk.py (hunk 4 above)
Comment (helmstedt) [reply login_flows/borgerdk.py]: In this case, no! Cause of my copy/paste errors I think.

**Comment by helmstedt** [comment]
Fixes and a new script should be ready now.

**Comment by Hundter** [comment]
Very nice, merging!

**Pull request merged**